### PR TITLE
Bug/duplicates

### PR DESCRIPTION
**Pull request by jackfrancismurphy**

### CODE REVIEW

**Comment by JackDanielHarding** [comment]
Screenshot 2022-05-17 at 16 50 27

Screenshot 2022-05-17 at 16 50 18

There is a bug in this code, I made my own test wiht the lion the witch and the wardrobe and when I guessed "The" it also unscrambled "Lion" and "Witch"

File: newdle_frontend.js [hunk 1]
```diff
@@ -52,29 +52,36 @@ function getHeadlines(headlines_json){
...
     if (headlineArrayLower.includes(PLAYER_GUESS)){
-        RESULT_PTAG.innerText = ""
-        const GUESS_POSITION = headlineArrayLower.indexOf(PLAYER_GUESS)
-        GUESS_BOX.value = ""
+        wordIndexes = getWordIndexes(headlineArrayLower,PLAYER_GUESS)
         var updatedHeadline = document.getElementById("presented_headline").innerText.split(" ")
-        updatedHeadline[GUESS_POSITION] = headlineArray[GUESS_POSITION]
-
-        // below we are changing the inner text of the <h2> element on the html file.  
+    for (i of wordIndexes){
```
[the remark this answers is not in the document]
Comment (jackfrancismurphy) [reply newdle_frontend.js:63]: *bangs head against wall*

[the remark this answers is not in the document]
Comment (jackfrancismurphy) [reply newdle_frontend.js:63]: I accidentally did this: [Merge branch 'main' into bug/duplicates](https://github.com/jackfrancismurphy/newdle-frontend/pull/42/commits/f9cc389b0259e1ab04ead2e6171c3eaccc4a898f)

Does that matter that it's a different commit now? What do I do? 😵

**Comment by jackfrancismurphy** [comment]
I've just had another notification for this, I thought I had done so, no? x

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Review by jackfrancismurphy — COMMENTED** [review]
The 'for' loop is now properly indented

**Review by JackDanielHarding — APPROVED** [review]

**Pull request merged**